python/cudf/cudf/io/json.py

```diff
@@ -97,7 +112,7 @@ def read_json(
             lines,
             compression,
             byte_range,
-            engine == "cudf_experimental",
+            engine == "cudf_legacy",
```

Comment:
This is switching from the new reader to the old reader. Should this be:
```suggestion
            engine == "cudf",
```

Reply:
The cython API changes too in this PR, that's why it has to be `engine=='cudf_legacy'`

Reply:
Wow, my mistake. I misread this as a parameter assignment `engine="cudf_legacy"` and not an equality check.